backupninja/files/rsync.conf

```diff
@@ -5,6 +5,16 @@
 # rsync handler file
 #
 
+#scheduling
```

Comment:
I think the same configuration apply to all backends, not only rsync. Can you please check this and apply for others?